pkg/scanners/scannerv4/scannerv4.go

```diff
@@ -229,6 +229,18 @@ func (s *scannerv4) GetVulnerabilities(image *storage.Image, components *types.S
 	ctx, cancel := context.WithTimeout(context.Background(), scanTimeout)
 	defer cancel()
 
+	// Store the index report from external scanners. Note that this will
+	// use some time from the scan timeout.
+	imageScanScannerVersion, err := pkgscanner.DecodeVersion(image.Scan.ScannerVersion)
+	if err != nil {
+		log.Warnf("Failed to decode image scan scanner version: %v", err)
+	} else {
+		err := s.scannerClient.StoreImageIndex(ctx, digest, imageScanScannerVersion.Indexer, v4Contents)
+		if err != nil {
+			log.Warnf("Failed to store external index report: %v", err)
+		}
+	}
```

Comment:
Maybe this should be a go routine instead? The `GetVulnerabilities()` call doesn't rely on the result of this anyway, and using a go routine would avoid using up the scan timeout. What'd yall think?